scripts/cloud-release.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+source ./scripts/cloud-utils.sh
+export RELEASE_ROLE_NAME=CodebuildRelease
+export RELEASE_PROFILE_NAME=AmplifyAPIPluginRelease
+export RELEASE_PROJECT_NAME=Release
+
+function release {
+  branch_name=$(git branch --show-current)
```

Comment:
One thing to note that this publish process will need an additional `git push` for the release branch to remote as the codebuild does not get the local branch. If we decide to do it separately, it should be added as an additional step in runbook

Reply:
The current DX I have in mind is that: 
1. Create a PR from `main` to `release`, review and merge it in.
2. Run `yarn release-codebuild` from the `release` branch.
3. (Codebuild) Runs `publish:release` script in our `package.json`. I believe this creates and pushes the release commit.
4. (Codebuild) Syncs the main branch with release using the `postpublish:release` script in our `package.json`.

Are you referring to the part 1 or part 3?

Reply:
I am referring to 1. I forgot a PR is merged before running the release script. The steps LGTM.